pkpdapp/pkpdapp/models/dataset.py

```diff
@@ -184,9 +184,9 @@ def replace_data(self, data: pd.DataFrame):
 
             has_observation = observation != "."
             is_observation_event = True
-            if event_id:
-                event_id = int(event_id)
-                is_observation_event = event_id == 0
+            if event_id is not None:
+                event_id_int = int(event_id)
+                is_observation_event = event_id_int == 0
```

Comment:
The tests are failing when `event_id` is an empty string, so I think this should be something like

```suggestion
            try:
                event_id_int = int(event_id)
                is_observation_event = event_id_int == 0
            catch ValueError:
                is_observation_event = has_observation
```

Reply:
ok, this looks good. How about the other if statement below?

Reply:
I think that needs `try/except` too. Otherwise the test are failing for a CSV that has empty EVID values.

Reply:
Oh, that suggestion shoould have been `except`, not `catch`.